### PR TITLE
Hello world project

### PR DESCRIPTION
**Pull request by TamaraDimov**
- Create new repository - Hello-world
- Create linters file in master branch
- Add a .gitignore file in master branch
- Create read me file
- Make new branch Hello-world-project
- Clone the repository to the local repo
- Add index.html , style.css and test.md files in the Hello-world-project branch
- Commit and push
- Create pull requests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,85 +52,34 @@ After you're finished please remove all the comments and instructions!
...
-
-<details>
-  <summary>Server</summary>
-  <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
-  </ul>
-</details>
-
-<details>
-<summary>Database</summary>
```
Comment (iambenkis) [inline README.md]: - [ ] Great work using a correct GitHub flow :100: , however, the feature branch has some conflict you need to solve. Kindly solve them. For more information about conflicts check this [link](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github)

**Review by iambenkis — CHANGES_REQUESTED** [review]
Hi @TamaraDimov , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



Screenshot from 2022-10-20 09-38-51

## Highlights :green_circle: 


- Correct Gitflow :heavy_check_mark: 
- PR has a good title and summary :heavy_check_mark: 
- Descriptive README.md file.  :heavy_check_mark: 

## Required Changes ♻️

- [ ] Great work using a correct GitHub flow, however, the feature branch has some conflict you need to solve. Kindly solve them. For more information about conflicts check this [link](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github)

Screenshot from 2022-11-29 23-55-41

- [ ] Good job creating initializing startup files :+1: , but if you review at the bottom of the pull request the linters have not been checked. Kindly install the linters by following the steps in this [documentation.](https://github.com/microverseinc/linters-config/tree/master/html-css#webhint). You will understand more about linters by reading carefully. :100:

- [ ] You follow most of the project requirements, unfortunately you forget to create ```.gitignore```. Kindly create it and make sure that it contains ```node_modules``` and ```test.md``` . I will suggest you to read more about the use of the ```.gitgnore``` if needed :+1:   

_Check the comments under the review._

## Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by TamaraDimov** [comment]
@iambenkis 
Hi Benjamin Kisenge, I did the following changes:
1. Properly install the linters 
2. Include node_modules and test.md in the .gitingnore file
3. Resolve the conflicts 

 Can you please review it again?
 Thank you so much

**Comment by iambenkis** [comment]
Before you merge and pull request you should first ask for a code review. You are not allowed to merge until the you get an **APPROVED STATUS** from a code reviewer !
You should ask for a code review first !

**Comment by TamaraDimov** [comment]
@iambenkis 
Sorry for my mistake. Can you please tell me how can I fix it?

**Comment by iambenkis** [comment]
Okay, in this case you should use the same PR and ask for another code review.

**Comment by TamaraDimov** [comment]
@iambenkis
Benjamin,
can you please help me to correctly revert the file? I already revert it but it came out without html and css file in it.

**Comment by iambenkis** [comment]
You should ask for a review with the link of this PR

**Comment by TamaraDimov** [comment]
@iambenkis
It was not possible to use the same PR so I created a new PR with this link [(https://github.com/TamaraDimov/Hello-world/pull/4)]
I revert the merged files and I correct all the things that you pointed out:
-  Add node_modulse on .gitignore
- Fix the linters by correcting the path

I learned a lot during the process and i want to thank you for the feedback and fast response

**Pull request merged**